.github/workflows/aws-production.yml

```diff
@@ -19,6 +19,7 @@ on:
 env:
   AWS_REGION: us-east-1                                        # set this to your preferred AWS region, e.g. us-west-1
   BUCKET_NAME: docs.oak.tech
+  CF_DISTRIBUTION_ID: E1AES8IDWU4SMA
```

Comment:
any concern having this value exposed?

Reply:
I don't think so. The distribution id is useless without the corresponding iam permissions.